test/parallel/test-cluster-disconnect.js

```diff
@@ -72,7 +73,11 @@ if (cluster.isWorker) {
     let online = 0;
 
     for (let i = 0, l = workers; i < l; i++) {
-      cluster.fork().on('listening', common.mustCall(() => {
+      cluster.fork().on('listening', common.mustCall((address) => {
+        if (serverPorts.indexOf(address.port) === -1) {
```

Comment:
>Will we be backporting PRs that fix potentially flaky tests to v4.x?

Almost certainly not. `! serverPorts.includes(address.port)` should be fine here.